src/components/VDatePicker/VDatePicker.js

```diff
@@ -288,6 +287,22 @@ export default {
       this.inputDay = parseInt(value.split('-')[2], 10)
       this.emitInput(this.inputDate)
     },
+    mouseEnter (value) {
+      this.inputYear = parseInt(value.split('-')[0], 10)
+      this.inputMonth = parseInt(value.split('-')[1], 10) - 1
+      this.inputDay = parseInt(value.split('-')[2], 10)
+      const output = this.multiple
+        ? (
+          this.value.indexOf(value) === -1
+            ? this.value.concat([value])
+            : this.value.filter(x => x !== value)
+        )
+        : value
+      this.$emit('mouseover', output)
```

Comment:
Why not just emit value? Also use the same event name (mouseenter)

Reply:
Oh i see what you did here, so probably using dom events names is not the best idea, still I think that there should be 2 events ("*enter" and "*leave"), both sending the value, but I'd change the naming. Maybe `date-mouse-enter` and `date-mouse-leave` or something similar?

Reply:
The idea was to keep the implementation simple with just @mouseover dom event that returns the date value or null (while leaving). But yes, I agree there should be two events. Event names date-mouse-enter and date-mouse-leave sounds good to me.